### DISCUSSION TITLE
Bug with module contact your customers

### DISCUSSION THREAD

**Discussion by moulti-pass** [PrestaShop/PrestaShop#36121] [Q&A]
Good morning,

I installed and configured the "Contact your customers" v2.0.0 module - by Prestashop but it does not seem to work because no emails are sent. The cron task generated by the module returns with an error: http://www.stagingalive.com/modules/followup/cron.php?secure_key=28WIS2DZXNMQYOJJ

This is my site: https://www.stagingalive.com/

Do you have any idea what this could be?

My version of Prestashop: 1.7.8.7
PHP version: 7.2

Thanks in advance,

Pierre

**Comment by SharakPL** [comment]
First of all I strongly recommend upgrading your shop to at least 1.7.8.11 as your current version has been one of the targets for hackers.

I assume you're using [ps_reminder-v.2.0.0](https://github.com/PrestaShop/ps_reminder/releases/tag/v2.0.0). Looks like it does have a bug with linking it to its PS 1.6 equivalent module
https://github.com/PrestaShop/ps_reminder/blob/43af077e3929967704e1e3b69d054ba84d6c975e/ps_reminder.php#L752

Problem is both modules have been archived for years so there's no way to fix it. You should stop using this module and find another with this functionality.
